### PR TITLE
Task/upgrade graphql 15 5 3

### PR DESCRIPTION
**Pull request by luan-asym**
# Summary

> Updates dependencies to most recent version

## Changelog

- Update to graphql 15.5.3 and accompanying changes
- Update other dependencies with no breaking changes
- Update the mention of PHX Tools to node-fhir-server-core

## Additional Notes

- `@asymmetrik/sof-strategy` should have a new package (1.1.5), but is not yet released. This will be added in a future PR

### CODE REVIEW

File: packages/fhir-gql-schema-utils/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+
+## [1.1.5](https://github.com/Asymmetrik/node-fhir-server-core/compare/@asymmetrik/sof-graphql-invariant@1.1.4...@asymmetrik/sof-graphql-invariant@1.1.5) (2021-09-16)
+
+### Update Dependency
+* Update graphql to 15.5.3
```
Comment (ekoon121) [inline packages/fhir-gql-schema-utils/CHANGELOG.md:10]: 15.6.0?

File: packages/fhir-gql-schema-utils/package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "test": "jest"
   },
   "devDependencies": {
-    "graphql": "^14.0.2"
+    "graphql": "^15.5.3"
```
Comment (ekoon121) [inline packages/fhir-gql-schema-utils/package.json:17]: 15.6.0?

File: packages/sof-graphql-invariant/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,13 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.0.0](https://github.com/Asymmetrik/node-fhir-server-core/compare/chore/convert-repo-to-monorepo...task/upgrade-graphql-15-5-3) (2021-09-16)
+
+### Update Dependency
+* Update graphql to 15.5.3
```
Comment (ekoon121) [inline packages/sof-graphql-invariant/CHANGELOG.md:9]: 15.6.0?

File: packages/sof-graphql-invariant/package.json [hunk 4]
```diff
@@ -1,18 +1,22 @@
...
   "author": "Robert-W <rwinterbottom@asymmetrik.com>",
+  "contributors": [
+    "Eugene Koon <ekoon@asymmetrik.com",
+    "Luan Tran <luan@asymmetrik.com"
+  ],
   "main": "index.js",
   "dependencies": {
     "@asymmetrik/sof-scope-checker": "^1.0.7",
-    "graphql": "^14.0.2"
+    "graphql": "^15.5.3"
```
Comment (ekoon121) [inline packages/sof-graphql-invariant/package.json:19]: 15.6.0?

**Comment by luan-asym** [comment]
need to change the branch name before merge

**Pull request merged**